From b56098bdd8b7645ae98bc57840d1544e067dba13 Mon Sep 17 00:00:00 2001
From: Vitaly Margolin <vmargolin@habana.ai>
Date: Thu, 22 May 2025 12:31:43 +0300
Subject: [PATCH] [SW-229877] g3: serdes: move serdes partition in flash

Move Serdes partition in flash from address
1C0000 to 1D0000 in CI-friendly way (after all CIs)
old constant will be removed.

Change-Id: I11161a76ef537bd5d8e0fc4b0c879b847fd0750e
Signed-off-by: Vitaly Margolin <vmargolin@habana.ai>
---

diff --git a/gaudi3/gaudi3_mem_addr_dt.h b/gaudi3/gaudi3_mem_addr_dt.h
index 93cf26a..a62e027 100644
--- a/gaudi3/gaudi3_mem_addr_dt.h
+++ b/gaudi3/gaudi3_mem_addr_dt.h
@@ -318,6 +318,7 @@
 #define RMA_2_PARTITION_SIZE				10000
 
 #define SPI_SERDES_CONFIG_OFFSET			1c0000
+#define SPI_SERDES_CONFIG_OFFSET_NEW			1d0000
 #define SPI_SERDES_CONFIG_SIZE				10000
 #define SPI_FLASH_HL338_LFS_OFFSET			c00000
 #define SPI_FLASH_HL338_LFS_SIZE			400000
